content/influxdb/clustered/write-data/best-practices/data-lifecycle.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Data ingest lifecycle best practices
+seotitle: InfluxDB best practices for data ingest lifecycle management
+description: >
+  Best practices for managing the lifecycle of data ingested into InfluxDB.
+menu:
+  influxdb_clustered:
+    name: Data ingest lifecycle
+    weight: 201
+    parent: write-best-practices
+---
+
+Data ingested into InfluxDB must conform to the retention period set on the database.
+Any data which lies outside of this retention period is no longer considered for retrieval by a query, but can have references maintained in object storage or the Postgres catalog, causing an increase in operational overhead and drive higher costs.
+
+In order to reduce these factors, it is important to manage the lifecycle of ingested data. You can achieve this by following the best practices below:
+
+- Use a retention policy which is appropriate for your requirements.
+- Tune the garbage collector cutoff period to ensure that data is removed in a timely manner.
+- Enable versioning in your object store and use lifecycle rules to expire old data.
+
+### Garbage collector tuning
+
+Once data falls outside of the retention period it can now be removed by the garbage collector service.
+
+There are two variables which tune this process:
+
+- `INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF`: defines when objects are deleted from object storage. The default is `30d`.
+- `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: defines when a row can be deleted from the catalog. The default is `30d`.
```

Comment:
I reckon these could benefit from being a bit more explicit here, so the interaction is clearer. Perhaps something like this:

```suggestion
- `INFLUXDB_IOX_GC_OBJECTSTORE_CUTOFF`: the age at which a parquet file not referenced in the catalog becomes eligible for deletion from object storage. The default is `30d`.
- `INFLUXDB_IOX_GC_PARQUETFILE_CUTOFF`: defines how long to retain the row referencing a parquet file in the catalog once marked for deletion. The default is `30d`.
```